JASP-Engine/JASP/R/reliabilityAnalysisBayesian.R

```diff
@@ -436,9 +439,23 @@ reliabilityBayesian <- function(jaspResults, dataset, options) {
       tb <- cbind(tb, newtb)
     }
 		itemTable$setData(tb)
+		
+		if (!is.null(unlist(options[["reverseScaledItems"]]))) {
+		  itemTable$addFootnote(sprintf(ngettext(length(options[["reverseScaledItems"]]),
```

Comment:
you're mixing tabs and spaces here. Please only use spaces in R code.

Reply:
okay. So I should restructure the R-files to only have spaces instead of tabs? I know it is a mess right now, because I didn't start out with that premise :( How can I see where I used tabs and where I used spaces?